models/metrics/updater.py

```diff
@@ -0,0 +1,48 @@
+import glob
```

Comment:
No need to put this script in this package

Reply:
Made changes. Please review